presidio-analyzer/Pipfile

```diff
@@ -10,6 +10,7 @@ tldextract = "*"
 numpy = "==1.19.3"
 flask = "==1.1.2"
 pyyaml = "*"
+flasgger = "*"
```

Comment:
Should we restrict the version? How robust is this package in terms of backward compatibility?

Reply:
let's restrict, don't think they had breaking changes yet

Reply:
Done